src/adapters/vfs/system.js

```diff
@@ -57,6 +57,11 @@ const methods = (core, request) => {
       .then(({body}) => body);
 
   return {
+    capabilities: ({path}, options) => request('capabilities', {
```

Comment:
I believe this can be shortened to `capabilities: passthrough('capabilities'),`

Reply:
Indeed. The wrapper function was designed for cases like this :)